cmd/crank/xpkg/install.go

```diff
@@ -50,27 +50,44 @@ const (
 	errKubeClient    = "failed to create kube client"
 )
 
-// InstallCmd is exported so that it can be re-used by the beta xpkg subcommand.
+// installCmd installs a package.
+type installCmd struct {
+	// Arguments.
+	Kind  string `arg:"" help:"The kind of package to install. One of \"provider\", \"configuration\", or \"function\"." enum:"provider,configuration,function"`
+	Image string `arg:"" help:"The OCI image name of the package to install."`
+	Name  string `arg:""  optional:"" help:"The name of the new package resource in the Crossplane API. Derived from the package's OCI image name by default."`
```

Comment:
Updated to:

```
  [<name>]    The name of the new package in the Crossplane API. Derived from
              the package's OCI image name by default.
```

I prefer to keep "in the Crossplane API". "Inside Crossplane" is confusing to me.

It doesn't look like the name is derived from package metadata (I interpret that to mean crossplane.yaml). It's derived from the package's OCI image name.

This command feels really hard because there's two instances of "package name" to try to distinguish. 😦

Reply:
Went with "package" instead of "image", and examples to illustrate.